### ISSUE TITLE
Microsoft Visual C++ 2017-2019 Redistributable (x64) - Throw exception when using the latest version: 14.27

### ISSUE THREAD

**Issue by raychan3690** [AlturosDestinations/Alturos.Yolo#159]
Alturos.Yolo version: 3.0.6-alpha
Redistributable (x64) version: 14.27

Is there anyone knows how to solve this error? It works fine with 14.22 c++ redistributable, but it doesn' work on the latest version:14.27. 
I have looked through all the posts that have similar problems, but it doesn't help on the issue.
![image](https://user-images.githubusercontent.com/36153411/98224732-cc663700-1f8e-11eb-9a42-3c5ec075bcb2.png)

**Comment by graiSoft1982** [comment]
Add

 { @"Installer\Dependencies\VC,redist.x64,amd64,14.27,bundle", "Microsoft Visual C++ 2015-2019 Redistributable (x64)" }

 distribution to DefaultYoloSystemValidator.cs

**Comment by raychan3690** [comment]
Thx @graiSoft1982 I solve the problem by adding your code and building the nugget package from the source myself. 
But, I am curious. Does this mean if there is any new release of the c++ redistributable, I would need to add it and regenerate a package?

**Comment by graiSoft1982** [comment]
Yes. In this solition. Recommend to use regular expression to detect min ver of cpp redist and will work with future versions.
